libs/ledger-live-common/src/device-core/managerApi/repositories/ManagerApiRepository.ts

```diff
@@ -0,0 +1,38 @@
+// To refactor: use own entities
+import { DeviceVersionEntity } from "../entities/DeviceVersionEntity";
+import { FinalFirmware, OsuFirmware } from "../entities/FirmwareUpdateContextEntity";
+import { IdEntity } from "../entities/IdEntity";
+
+// Used to name interfaces like this "Port"
+export interface ManagerApiRepository {
```

Comment:
[ASK] What about having different classes as this repository seems to get different kind of data?

Reply:
It makes sense but in the current stack without a dependency injection system I think it will be too many parameters to pass to a given single use case (all the repositories). Or is there an alternative ?

Reply:
Yes you could combine repositories in a use case but it may be complex to implement due to the current code base